modules/administration/pages/troubleshooting/tshoot-logintimeout.adoc

```diff
@@ -44,6 +44,8 @@ web.session_database_lifetime = <Timeout_Value_in_Seconds>
 . Save and close the file.
 . Open [path]``/srv/tomcat/webapps/rhn/WEB-INF/web.xml`` and add or edit this line to include the new timeout value in minutes:
 +
+// FIXME 2023-12-12, ke: check /srv ^^^
```

Comment:
correct. Path is now `/usr/share/susemanager/www/tomcat/webapps/rhn/WEB-INF/web.xml` , but it should be somewhere else in `/srv`: this is because in uyuni container changes in `/usr/share/susemanager/` are not persisted. I'll open an issue for it, we need somehow to change this path and then re-change documentation

Reply:
ok, thanks for feedback!

Reply:
to track this we now have https://github.com/SUSE/spacewalk/issues/23204.  Thus I think this is ready to be merged.